README.md

```diff
@@ -14,6 +14,134 @@ bower install d2l-inputs
 
 ## Usage
 
+### d2l-input-checkbox and d2l-input-checkbox-spacer
+
+<img src="/images/checkbox.gif?raw=true" width="350">
+
+Include the [webcomponents.js](http://webcomponents.org/polyfills/) "lite" polyfill (for browsers who don't natively support web components), then import `d2l-input-checkbox.html` and `d2l-input-checkbox-spacer.html`:
```

Comment:
The bit about including the polyfill can move outside of the individual component sections.